src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -49,6 +49,27 @@ To view the logs for a build, scroll to the build checks at the bottom of the PR
 
 ![Screenshot of the build checks section of the PR page, with arrow pointing to primary link](./images/ci-build-logs.png 'Link to the build details with logs')
 
+## Pacman Package Cache
+
+When buildkite receives a webhook to build and test a PR, the buildkite agents pulls down the code and starts building a docker image based off the Dockerfile in that branch.
+
+In the process of building our docker image, serveral [pacman](https://wiki.archlinux.org/index.php/pacman) packages are downloaded from public mirrors (if that specific [layer](https://docs.docker.com/storage/storagedriver/#images-and-layers) itself is not cached). Because these packages are downloaded numerous times, across multiple PR's, caching them locally can speed up build times and save network bandwidth.
```

Comment:
There have also been instances where the public mirrors have changed the signature on a package, but because we date lock our packages we are still expecting an older signature. Caching these package **_may_** alleviate this issue as well.